pkg/config/analysis/incluster/controller.go

```diff
@@ -113,7 +113,7 @@ func (c *Controller) Run(stop <-chan struct{}) {
 			log.Debugf("finished enqueueing all statuses")
 		case <-stop:
 			t.Stop()
-			break
+			return
```

Comment:
This is a bug, but how is it related to CPU high?

Reply:
it's an infinite looping, end up with 100% cpu on selectgo. Once stop is closed

Reply:
I mean stop can be closed only when the process exit?